docs-compose/overwatch_docs/content/DataEngineer/Productionizing.md

```diff
@@ -55,6 +56,240 @@ created each run but it will be replaced on each subsequent run. If you would li
+
+1. **Type** - Jar
+2. **Main class** - com.databricks.labs.overwatch.pipeline.RestoreRunner
+3. **Parameters for the Job** - Same as mentioned above for `Restore through Databricks Notebook`
+
+![Restore_Job](/images/DataEngineer/restore_job.png)
+
+#### Validation Functionality
+This is a special module in Restore  Process. By default, it is part of Restore.process() but if user want to run it separately they can run it separately
+to check whether the configuration they would provide for Restore process, are correct or not.
+
+Below is the code snippet for the validation function
+
+```scala
+import com.databricks.labs.overwatch.pipeline.Restore
+val sourcePrefix = "/mnt/overwatch_playground/721-snap/full"
+val targetPrefix = "/mnt/overwatch_playground/721-snap/restore"
+Restore.isValid(sourcePrefix,targetPrefix)
+```
+
+Upon running the above function you will get proper output whether the input parameters you would be using for Restore process,passed the validation check or not. One example of the same is as below:
+
+```
+SourcePrefix Path Exists.
+Target Path /mnt/overwatch_playground/721-snap/restore is Empty and is Suitable for Restore Process
+Validation successful.You Can proceed with Restoration process
+import com.databricks.labs.overwatch.pipeline.Restore
+res43: Boolean = true
+```
+
+
+### Migration Process
+Migration Process is same as Snapshot process with some added Functionality. Below are the steps involved in Migration Process:
+
+1. Stop Overwatch jobs  (Need to be done by User)
+2. Migrate all old data to new location. (Would be done by OW Migration Process) 
+3. Update Config with new storage location (Would be done by OW Migration Process)
+4. Delete Old Database  (Would be done by OW Migration Process)
+5. Resume Overwatch jobs (Need to be done by User)
+
+##### Migration through Databricks Notebook
+The code snippet to run the Migration process is as below:
+```scala
+import com.databricks.labs.overwatch.pipeline.Migration
+val sourceETLDB = "ow_bronze_migration_etl"
+val migrateRootPath = "/mnt/overwatch_playground/721-snap/migration"
+val configPath = "abfss://overwatch-field-playground@overwatchglobalinternal.dfs.core.windows.net/sourav/configs/721-migration"
+val tablesToExclude = "notebook_silver:notebook_gold"
+Migration.process(sourceETLDB,migrateRootPath,configPath)
+```
+We need below parameters to run the Migration process through Databricks Notebook:
+
+| Param           | Type     | Optional   | Description                                                                                                                           |
+|-----------------|----------|------------|---------------------------------------------------------------------------------------------------------------------------------------|
+| sourceETLDB     | String   | No         | Source Database Name for which Migration need to be done.                                                                             |
```

Comment:
Change description to: "Source Database Name to migrate. Note that you can only use the Overwatch ETL database"

Reply:
Done